Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN dotnet restore
 
 # Copy everything else and build
 COPY . ./
+RUN apt-get update && apt-get install npm -y
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5434886)